pyproject.toml

```diff
@@ -37,6 +37,7 @@ filterwarnings = [
   "ignore:The 'asyncio_mode' default value will change to 'strict' in future:DeprecationWarning",
   'ignore::scipy.optimize._optimize.OptimizeWarning',
   # Plotting related warnings.
+  'ignore:The old plotting backend is deprecated:scipp.VisibleDeprecationWarning',
```

Comment:
You should now add a new entry in #2864 to remove this entry, once the "Refactor final uses of scipp plotting in docs" item is resolved.

Reply:
This is not about docs but about tests. So this setting needs to stay until we have *removed* the old code and tests.